src/test/java/it/pagopa/pn/delivery/svc/search/MultiPageSearchTest.java

```diff
@@ -27,7 +27,7 @@
 import java.util.Optional;
 import java.util.concurrent.ConcurrentHashMap;
 
-class MultiPageSearchTest {
+public class MultiPageSearchTest {
```

Comment:
Trasformarla in una classe di utility in un suo file.

Reply:
Spostata la classe innestata